product_template_reassign/wizard/reassign_products.py

```diff
@@ -0,0 +1,76 @@
+
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
+
+from openerp import fields, models, api
+from itertools import product
+
+
+class ReassignProducts(models.TransientModel):
+    _name = "reassign.products"
+    _rec_name = "products"
+    products = fields.One2many('reassign.product.lines',
+                               'reassign', string='Attributes Products')
+
+    @api.multi
+    def _product_by_variants(self, variants, products):
+        set_var_ids = set(variants)
+        for product in products:
```

Comment:
@mikelarre esta línea hace fallar el travis ya que importas desde itertools el product, y aquí haces un `for` asignándolo a product. Puedes corregirlo, porfa.  :wink: